policy/resolver.go

```diff
@@ -909,6 +909,9 @@ func (cache *policyResolverCache) addCheckJob(ctx context.Context, check *explor
 	// ^^
 
 	if len(check.Variants) != 0 {
+		// Its easier to deal with things if we dont mix query code ids with
+		// query mrns.
+		queryJob.QrId = check.CodeId
```

Comment:
I cant figure out the correct place to put this. There is other code that changes qrid from mrn to code id for checks, but I dont think it runs for these queries